Short patch against adesklets-0.4.10 that makes sure GTK theme change doesn't
make adesklets exit. Apply with:

cd /path/to/adesklets-0.4.10 && \
	patch -p1 < /path/to/adesklets_gtk-chtheme.diff
  
--- a/src/adesklets.c	2005-06-10 20:17:00.000000000 -0400
+++ b/src/adesklets.c	2005-08-17 01:25:43.000000000 -0400
@@ -2930,10 +2930,14 @@
 	    adesklets_images_reset_background(0);
 	break;
       case ClientMessage:
-	/* Since we register only one protocol (WM_WINDOW_DELETE),
-	   there is no need for further checks */
-	debug("Message from window manager: quitting!\n");
-	adesklets.user_quit_flag=adesklets.quit_flag=1;
+	/* Check for the "WM_WINDOW_DELETE" protocol */
+	if (ev.xclient.data.l[0] == XInternAtom(adesklets.display,
+						"WM_WINDOW_DELETE",
+						False)) {
+	  debug("Message from window manager: quitting!\n");
+	  adesklets.user_quit_flag=adesklets.quit_flag=1;
+	} else 
+	  debug("Unknown client message sent.");
 	break;
       default:
 	/* Untreated cases */
